src/Compilers/CSharp/Portable/CommandLine/CSharpCompiler.cs

```diff
@@ -267,6 +267,28 @@ public override void PrintLogo(TextWriter consoleOutput)
             consoleOutput.WriteLine();
         }
 
+        public override void PrintLangVersions(TextWriter consoleOutput)
+        {
+            consoleOutput.WriteLine(ErrorFacts.GetMessage(MessageID.IDS_LangVersions, Culture));
+            var defaultVersion = LanguageVersion.Default.MapSpecifiedToEffectiveVersion();
+            var latestVersion = LanguageVersion.Latest.MapSpecifiedToEffectiveVersion();
+            foreach (LanguageVersion v in Enum.GetValues(typeof(LanguageVersion)))
+            {
+                if (v == defaultVersion)
+                {
+                    consoleOutput.WriteLine($"{v.ToDisplayString()} (default)");
+                }
+                else if (v == latestVersion)
+                {
+                    consoleOutput.WriteLine($"{v.ToDisplayString()} (latest)");
+                }
+                else if (v != LanguageVersion.Default && v != LanguageVersion.Latest)
```

Comment:
Yes, when v is LanguageVersion.CSharp6 for example.
I should have included an example of what this method prints:
```
1
2
3
...
7 (default)
7.1 (latest)
```
This check ensures that LanguageVersion.Default and LanguageVersion.Latest are skipped.